cpp/util/masterelection.cc

```diff
@@ -88,6 +99,8 @@ MasterElection::MasterElection(const shared_ptr<libevent::Base>& base,
       backed_proposal_(kNoBacking),
       is_master_(false) {
   CHECK_NE(kNoBacking, node_id);
+  is_master_gauge->Set(0);
+  participating_in_election_gauge->Set(0);
```

Comment:
One thing that's always bothered me with this approach to metrics: what if you have more than one election in the same process?